### ISSUE TITLE
multiple executions

### ISSUE THREAD

**Issue by tomassatka** [graphql-java-generator/graphql-maven-plugin-project#121]
I would have a question regarding multiple executions within one plugin definition.
My intention is to generate a server code for graphql that my service expose but also generate a pojo for objects consumed by kafka-connector based on graphql schema of source.

I tried to add multiple executions but the second execution is not done. 

```xml
<plugin>
  <groupId>com.graphql-java-generator</groupId>
  <artifactId>graphql-maven-plugin</artifactId>
  <version>1.18</version>
  <executions>
      <execution>
          <id>generate-a</id>
          <goals>
              <goal>generateServerCode</goal>
          </goals>
          <phase>process-resources</phase>
          <configuration>
              <packageName>foo.bar</packageName>
              <schemaFileFolder>src/main/resources/foo</schemaFileFolder>
              <generateJPAAnnotation>true</generateJPAAnnotation>
              <javaTypeForIDType>java.lang.String</javaTypeForIDType>
              ...
          </configuration>
      </execution>
      <execution>
          <id>generate-b</id>
          <goals>
              <goal>generatePojo</goal>
          </goals>
          <phase>process-resources</phase>
          <configuration>
              <packageName>foo.eee</packageName>
              <schemaFileFolder>src/main/resources/eee</schemaFileFolder>
              <mode>client</mode>
              ...
          </configuration>
      </execution>
  </executions>
</plugin>
```

when i add to second execution different output directory `<targetSourceFolder>./target/generated-sources/graphql-maven-plugin-bar</targetSourceFolder>` then i see that both codes get generated but however then i get whole bunch of errors complaining about duplicated code
```
[ERROR] /C:/Develop/IdeaProjects/sixsense/packages/observability-service/target/generated-sources/graphql-maven-plugin-hypertrace/com/graphql_java_generator/annotation/GraphQLInputParameters.java:[19,9] duplicate class: com.graphql_java_generator.annotation.GraphQLInputParameters
```
caused by multiplication of `target/generated-sources/graphql-maven-plugin/com/graphql_java_generator/annotation` 

So basically the question is: are multiple executions supported? 
What would be your suggestion to implement multiple generations if the app is consuming N graphql apis and exposes N graphql apis?

**Comment by tomassatka** [comment]
I missed adding `<copyRuntimeSources>false</copyRuntimeSources>` to skip generation of runtime code. Adding it does the trick..
